client/sites-dashboard/components/sites-display-mode-switcher.tsx

```diff
@@ -34,11 +37,37 @@ const SiteSwitcherButton = styled( Button )<
 	};
 } );
 
+type SitesDisplayMode = 'tile' | 'list' | 'none';
+
+const PREFERENCE_NAME = 'sites-management-dashboard-display-mode';
+
 export const SitesDisplayModeSwitcher = () => {
+	const dispatch = useDispatch();
+
+	const onDisplayModeChange = ( value: SitesDisplayMode ) => {
+		dispatch( savePreference( PREFERENCE_NAME, value ) );
+	};
+
+	const displayMode: SitesDisplayMode = useSelector( ( state ) => {
+		if ( ! hasReceivedRemotePreferences( state ) ) {
+			return 'none';
+		}
+
+		return getPreference( state, PREFERENCE_NAME ) ?? 'list';
```

Comment:
Oh something else that can be either fixed up here or in another PR: the tile view should be the default.
```suggestion
		return getPreference( state, PREFERENCE_NAME ) ?? 'tile';
```

Reply:
Oops, missed that.